docs/source/migrationGuide2.0.rst

```diff
@@ -0,0 +1,68 @@
+.. _lbl_migration_guide2.0:
+
+Migration Guide 2.0
+-------------------
+
+With the release of SimpleITK 2.0 there are changes to the interface and to the behaviour of some methods and classes. The intent of this document is to provide guidance on how to modify code so that it is compatible with SimpleITK 2.0.
+
+DICOM reading
++++++++++++++
+
+Changes to DICOM reading, due to SimpleITK's move from ITK 4.x to 5.x. Photomertic interpretation, tag 0028|0004, is now taken into account when reading the image.
+
+   1. MOCHOROME1: previously intensity values loaded as is, now they are inverted. If you previously inverted the intensities yourself you will need to remove that code.
+   2. Color images (e.g. YBR_FULL_422). Previously channels loaded as is, now they are converted to RGB. If you previously used the photometric interpretation tag value to perform color space conversions outside of SimpleITK you will have to modify your code. For example, previously the first channel for an image with photometric interpretation of YBR_FULL luminance and could be used as the grayscale version of the image. Now you will need to use all three channels to compute the image luminance:
```

Comment:
Colon after 'Color images (e.g. YBR_FULL_422)'

Also, my preference would be to avoid 2nd person 'you' if possible.

Reply:
Updated.